### PR TITLE
setup multiple tables

### PR DESCRIPTION
**Pull request by MarkoKermi**
In this PR I did the following requirements: 

- [ ] Create a table named `owners`
- [ ] Create a table named `species`
- [ ] Modify `animals` table
- [ ] Insert data for table `owners` and for `species`
- [ ] I modify inserted animals so it includes the `species_id` value.
- [ ] I modify inserted animals to include owner information `owner_id`

- What animals belong to Melody Pond? 
Screenshot 2023-03-29 at 11 47 37

- List of all animals that are pokemon (their type is Pokemon).
Screenshot 2023-03-29 at 12 32 49

- List all owners and their animals, remember to include those that don't own any animal.
Screenshot 2023-03-29 at 12 06 24

- How many animals are there per species?
Screenshot 2023-03-29 at 12 34 35

- List all Digimon owned by Jennifer Orwell.
Screenshot 2023-03-29 at 12 07 02

- List all animals owned by Dean Winchester that haven't tried to escape.
Screenshot 2023-03-29 at 12 09 18

- Who owns the most animals?
Screenshot 2023-03-29 at 12 08 25

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -16,3 +16,12 @@ select avg(weight_kg) from animals;
 select neutered, count(neutered) as result from animals group by neutered order by result desc limit 1;
 select species, MAX(weight_kg) as max_weight, MIN(weight_kg) as min_weight from animals group by species;
 select species,avg(escape_attempts) from animals where date_of_birth between '1900-01-01' and '2000-12-31' group by species;
+
+
+SELECT name as animal_name, full_name FROM animals A JOIN owners O ON A.owner_id = O.id WHERE full_name = 'Melody Pond';
+SELECT A.name as animal_name, S.name as specie_type FROM animals A JOIN species S ON A.species_id = S.id WHERE S.name = 'Pokemon';
+SELECT full_name, name as animal_name FROM animals A RIGHT JOIN owners O ON A.owner_id = O.id ; 
+SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT full_name, A.name FROM animals A JOIN owners O ON A.owner_id = O.id WHERE full_name = 'Jennifer Orwell';
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this query, but please kindly ensure you list only `all Digimon owned by Jennifer Orwell.`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @MarkoKermi , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by MarkoKermi** [comment]
The changes that I did based on second review: 

- List all Digimon owned by Jennifer Orwell.
Screenshot 2023-03-29 at 13 10 10

**Review by jerryowusu — APPROVED** [review]
Hi @MarkoKermi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
:heavy_check_mark: Nice PR summary
:heavy_check_mark: Great effort so far


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**